src/components/Checkbox/index.js

```diff
@@ -10,7 +10,7 @@ const StyledCustomCheckbox = styled.span`
     border: 2px solid ${colors.black};
     height: 14px;
     width: 14px;
-    transition: 150ms ease-in;
+    transition: 30ms ease-in;
```

Comment:
a 30ms transition, is that even noticeable?

Reply:
oh yeah 30ms is really small, it seemed really slow before